tests/test_actions.py

```diff
@@ -607,6 +708,10 @@ def test_can_be_instantiated(self):
         assert isinstance(scl3, SaveConsoleLog)
         assert isinstance(scl4, SaveConsoleLog)
 
+    def test_implements_protocol(self):
+        r = SaveConsoleLog("")
+        assert isinstance(r, Performable)
```

Comment:
If you don't mind, can you add the `describe` method to `SaveConsoleLog` and `SaveScreenshot`? I forgot to add those methods when i added the Actions.

Reply:
Added